CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # tap2junit changelog
 
+### 0.1.6
+* support for parsing YAML blocks
+* Add option to override test suite name
+* Add support for TAP version 12
+* support TAP with nested tests
+* fix duration_ms to be milliseconds instead of seconds
```

Comment:
```suggestion
* Support for parsing YAML blocks
* Add option to override test suite name
* Add support for TAP version 12
* Support TAP with nested tests
* Fix duration_ms to be milliseconds instead of seconds
```

Reply:
Explicitly encode input file as utf-8